src/scripts/init_blockchain.sh

```diff
@@ -70,5 +83,11 @@ cleos push action eosio.token issue '["eosio", "100000000.0000 EOS", "memo"]\' -
 echo "=== lamington: init system contract ==="
 cleos push action eosio init '[0, "4,EOS"]' -p eosio@active
 
+sleep 2s
+
+cleos push action eosio setpriv '["eosio.msig",1]' -p eosio
+
+sleeps 2s
```

Comment:
Should this be
```
sleep 2s
```
?

Reply:
Sorry typo. Once again this was only for reliability but clearly it's not doing anything here so I've removed it